src/snyk/common/vscode/languageConsts.ts

```diff
@@ -1,7 +1,6 @@
 export const TYPESCRIPT_FILE_REGEX = new RegExp('\\.tsx?$');
```

Comment:
This doesn't belong to `vscode` folder. `src/snyk/common/constants/language.ts` would be a better place for it.